packages/manager/apps/ips/src/tracking.constant.ts

```diff
@@ -0,0 +1,20 @@
+export const LEVEL2 = {
+  EU: {
+    config: {
+      level2: '81',
+    },
+  },
+  CA: {
+    config: {
+      level2: '81',
+    },
+  },
+  US: {
+    config: {
+      level2: '81',
+    },
+  },
```

Comment:
Refer -> https://github.com/ovh/manager/blob/596170e7b496f367ad75ab8273e6308d27e7b7a3/packages/manager/apps/dedicated/client/app/at-internet.constants.js
```suggestion
  EU: {
    config: {
      level2: '85',
    },
  },
  CA: {
    config: {
      level2: '85',
    },
  },
  US: {
    config: {
      level2: '57',
    },
  },
```